managed/ui/src/components/universes/UniverseForm/ClusterFields.js

```diff
@@ -608,7 +609,7 @@ export default class ClusterFields extends Component {
     if (volumeDetail) {
       const deviceInfo = {
         volumeSize: volumeDetail.volumeSizeGB,
-        numVolumes: volumesList.length,
+        numVolumes: volumeCount,
```

Comment:
could you please update logic here as well https://github.com/yugabyte/yugabyte-db/blob/b6bc73a25aa519dd879409bb1e69235e42a822f8/managed/ui/src/redesign/universe/wizard/fields/DeviceInfoField/DeviceInfoField.tsx#L65